### PR TITLE
Use keycloak directly for authentication.

### PR DESCRIPTION
**Pull request by TreyE**
https://www.pivotaltracker.com/story/show/185093216

### CODE REVIEW

File: apps/console/src/app/tenant-config.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const consoleTenantConfig: TenantConfig[] = [
     host: 'localhost',
     application: 'console',
     tenant: 'me',
-    baseApiUrl: 'https://preprod-console-api.cme.openhbx.org',
+    baseApiUrl: 'http://localhost:3000',
```
Comment (nisanthyaganti9) [inline apps/console/src/app/tenant-config.ts:8]: @TreyE  - is this change intentional ?

Comment (TreyE) [reply apps/console/src/app/tenant-config.ts:8]: Yes, this is for local host testing, which you should be doing with docker.

**Review by nisanthyaganti9 — APPROVED** [review]

**Review by kristinmerbach — APPROVED** [review]
